Dockerfile

```diff
@@ -50,15 +50,15 @@ RUN python3 -m pip install -e git+https://github.com/kytos-ng/of_core@${branch_o
  && python3 -m pip install -e git+https://github.com/kytos-ng/mef_eline@${branch_mef_eline}#egg=kytos-mef_eline \
  && python3 -m pip install -e git+https://github.com/kytos-ng/of_multi_table@${branch_of_multi_table}#egg=kytos-of_multi_table \
  && curl -L -o /tmp/latest.zip https://github.com/kytos-ng/ui/releases/${release_ui}/latest.zip \
- && python3 -m zipfile -e /tmp/latest.zip  /usr/local/lib/python3.9/dist-packages/kytos/web-ui \
+ && python3 -m zipfile -e /tmp/latest.zip  /usr/local/lib/python3.11/dist-packages/kytos/web-ui \
  && rm -f /tmp/latest.zip
 
 # end-to-end python related dependencies
 # pymongo and requests resolve to the same version on kytos and NApps
-RUN python3 -m pip install pytest-timeout==2.1.0 \
- && python3 -m pip install pytest==7.2.0 \
- && python3 -m pip install pytest-rerunfailures==10.2 \
- && python3 -m pip install mock==4.0.3 \
+RUN python3 -m pip install pytest-timeout==2.2.0 \
+ && python3 -m pip install pytest==7.2.1 \
```

Comment:
@Alopalao, can we also have `pytest==8.0.1` here similar to kytos-ng projects and NApps? or did you encounter any other issue? It'd be great to have `pytest` 8+ on e2e tests too.

Reply:
Just upgraded it. I tested it with first test file and it is working fine with `pytest==8.1.1`

Reply:
Cool. Good to know.